src/actions/defineCallbacks.java

```diff
@@ -0,0 +1,46 @@
+//
+// Define your callbacks
+//
+
+/**
+ * Fetch the Entry Table lines for the list of given {@link Uid32}. If a line does not exist, there should be no entry in the returned map.
+ *
+ * @param uids the unique {@link Uid32}s used as line id
+ * @return a {@link Map} of {@link Uid32} to {@link EntryTableValue}
+ * @throws CloudproofException if anything goes wrong
+ */
```

Comment:
Those comments are really heavy. Do we really need them? In the JS part, there is no explanation of how callbacks work.